backend/controller/identity/dal/dal.go

```diff
@@ -0,0 +1,42 @@
+package dal
```

Comment:
Can you put this into `backend/controller/identity/internal/dal` please? I'm doing that for all other services at the moment.